lib/upgrader.js

```diff
@@ -23,6 +26,55 @@ module.exports = ({ argv }) => {
     `);
   }
 
+  replace({ argv });
+  refactor({ argv });
+};
+
+// Simple transformations involving global replace
+function replace({ argv }) {
+  const files = [ ...globByExtension('js'), ...globByExtension('html') ];
+  const replaces = [
+    // Address common aliases that have changed in A3
+    [ 'apos.tasks', 'apos.task' ],
+    [ 'apos.locks', 'apos.lock' ],
+    [ 'apos.versions', 'apos.version' ],
+    [ 'apos.images', 'apos.image' ],
+    [ 'apos.groups', 'apos.group' ],
+    [ 'apos.permissions', 'apos.permission' ],
+    [ 'apos.docs', 'apos.doc' ],
+    [ 'apos.utils', 'apos.util' ],
+    [ 'apos.attachments', 'apos.attachment' ],
+    [ 'apos.caches', 'apos.cache' ],
+    [ 'apos.files', 'apos.file' ],
+    [ 'apos.users', 'apos.user' ],
+    [ 'apos.urls', 'apos.url' ],
+    [ 'apos.migrations', 'apos.migration' ],
+    [ 'apos.templates', 'apos.template' ],
+    [ 'apos.videoFields', 'apos.videoField' ],
+    [ 'apos.pages', 'apos.page' ],
+    // No blanket replace for areas because we don't want to confuse the
+    // separate issue of linting the "apos.area" helper by renaming "apos.areas"
+    // to that. These are the individual likely offenders
+    [ 'apos.areas.getWidgetManager', 'apos.area.getWidgetManager' ],
+    [ 'apos.areas.richText', 'apos.area.richText' ],
+    [ 'apos.areas.plaintext', 'apos.area.plaintext' ],
+    [ 'apos.areas.walk', 'apos.area.walk' ]
```

Comment:
`apos.areas.isEmpty` should be included here.

Reply:
It looks like `apos.area.richText` still triggers the "Use the new area syntax" linter message. Clemens.